PocketKit/Sources/PocketKit/Article/ReadableViewModel/ReadableViewModel.swift

```diff
@@ -108,4 +112,36 @@ extension ReadableViewModel {
         let event = SnowplowEngagement(type: .general, value: nil)
         tracker.track(event: event, contexts)
     }
+
+    func webViewActivityItemsForSavedItems(item: SavedItem) -> [UIActivity] {
```

Comment:
```suggestion
    func webViewActivityItems(for item: SavedItem) -> [UIActivity] {
```

Reply:
omg I never knew you can do this 😲

Reply:
@cyndichin https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID166

Reply:
oh lol I meant suggest a change

Reply:
well, at least we're on the same page now :)

Reply:
@dskuza Fixed.